### ISSUE TITLE
textarea element needs title attribute or label for accessibility

### ISSUE THREAD

**Issue by iainsimmons** [RIP21/react-simplemde-editor#41]
According to [WCAG 2.0](https://www.w3.org/TR/WCAG20/#robust), `textarea` form controls (and just about every other interactive UI element) should have a _name_, _role_ and _value_. The `textarea` generated by `React.createElement` in your code does not give it a _name_.

Their [techniques page](https://www.w3.org/TR/WCAG20-TECHS/H91) indicates that the name comes from 

> `<label>` element associated with it or `title` attribute

See also their [examples for basic HTML techniques](https://www.w3.org/TR/WCAG20-TECHS/H91#H91-ex7).

Can I suggest you pass through a `title` prop to the `textarea`? Probably easier than adding a `label` and having to style that.

PS. I discovered this using the handy [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) tool.

**Comment by iainsimmons** [comment]
I couldn't see it in your code, but it looks like there are two `textarea` elements generated, though the second looks to be the CodeMirror one.

**Comment by jrsinclair** [comment]
@iainsimmons It appears that the second `textarea` element is coming from the SimpleMDE module, not this one. You may want to raise an issue there instead.

**Comment by RIP21** [comment]
This one seems to be fixed on our side. So I close this one.
